src/test/common/page/in-scotland.js

```diff
@@ -10,13 +10,13 @@ const PAGE_TITLES = {
 /**
  * Page object for I Live in Scotland page where the claimant is told they cannot apply if they live in Scotland.
```

Comment:
Minor - would it be worthwhile removing the `I Live in Scotland` words from this comment, it then still makes perfect sense but without something that tracks the name of the object?